### DISCUSSION TITLE
Marginalen neu konzipieren

### DISCUSSION THREAD

**Discussion by BillyTheKing** [lotta-schule/web#267] [Ideen]
- Funktion von Marginalen teils verschwommen
- Willkür der Anzeige von bestimmten Marginalen bei bestimmten Kategorien, ohne für User erkennbare Logik
- eigentliche Wichtigkeit von bestimmten Marginalen wie Vplan, Kalender überschattet
- unklare Trennung von User Bereichen: Mein Profil, Meine Beiträge, Meine Medien über das User Menü aufrufbar; Kalender und Vplan über die Marginalen, obwohl alles logischerweise in einem Nutzer Bereich erscheinen sollte
- Vorschlag: wie beim Admin Bereich, einen Nutzer Bereich mit allen spezifisch für/auf den Nutzer angepassten "Apps" einzurichten, der individualisierbar ist und eine größere Bedeutung bekommt

**Comment by ptitmouton** [comment]
Aktueller Stand: @BillyTheKing kümmert sich um Vorschläge, wie das in der UI aussehen könnte

**Comment by ptitmouton** [comment]
Kurze Gedankenstütze nach Besprechung mit @BillyTheKing damit wir das nicht vergesen

Banner wird standardmäßig auf eine sehr niedrige Höhe eingestellt, im Banner befinden sich ausschließlich Überschrift und (IconButton) Link zur Suche.
Höhe des Banners ist eine Darstellungsfrage und kann dementsprechend auch als eine Darstellungsoption konfiguriert werden (wie etwa Abstände)
(billy bitte Bild einfügen)


Persöhnlicher Bereich ist ein Avatar in der Navigation, der ein Untermenü (wie eine normale Subkategorie) öffnet.
(billy bitte Bild einfügen)

Adminbereich verhält sich analog zum öffentlichen Bereich.
(billy bitte Bild einfügen)

Marginalen gibt es nicht mehr. Stattdessen werden auf jeder Kategorieseite persönliche Widgets angezeigt, auf die der Nutzer Zugriff hat (funktionsmäßig etwa gleich zu den aktuellen Marginalen). Auf mobil wird es als Menü wie bisher die Marginalen geöffnet. Für nicht-angemeldete Nutzer gilt dasselbe, nur dass sie halt logischerweise nur öffentliche Sachen sehen.
Widgets sind auf jeder Kategorieseite sichtbar. Eine Beziehung Widgets <->  Kategorien entfällt.
(billy bitte Bild einfügen)

**Comment by BillyTheKing** [comment]
Hallo Zusammen, hier kommt mal was neues.
Weil das Design doch umfangreicher wird, habe ich die Vorschau interaktiv gelöst.
**Wichtig - ihr könnt nicht alles anklicken -> ein Klick auf eine beliebige Stelle zeigt euch, welche Elemente klickbar sind, wie bspw. die ersten 3 Marginalen, auf das Männchen zum Anmelden, in den Navigationsbereich (Wappen Icon), zu "Mein Bereich" über das Profilbild in der Navigation**
Bin gespannt auf euer Feedback.
[https://xd.adobe.com/view/5e6a5017-67ab-41d1-5ef9-e56bc77bff26-285b/?fullscreen](https://xd.adobe.com/view/5e6a5017-67ab-41d1-5ef9-e56bc77bff26-285b/?fullscreen )
